src/test/java/org/opentripplanner/street/model/edge/VehicleRentalEdgeTest.java

```diff
@@ -218,17 +282,63 @@ private void initEdgeAndRequest(
 
     vehicleRentalEdge = VehicleRentalEdge.createVehicleRentalEdge(vertex, RentalFormFactor.BICYCLE);
 
+    Set<String> bannedNetworks = banNetwork ? Set.of(station.getNetwork()) : Set.of();
+
     this.request =
       StreetSearchRequest
         .of()
         .withMode(mode)
         .withPreferences(preferences ->
           preferences
-            .withCar(car ->
-              car.withRental(rental -> rental.withUseAvailabilityInformation(useRealtime))
-            )
             .withBike(bike ->
-              bike.withRental(rental -> rental.withUseAvailabilityInformation(useRealtime))
+              bike.withRental(rental ->
+                rental
+                  .withUseAvailabilityInformation(useRealtime)
+                  .withBannedNetworks(bannedNetworks)
+              )
+            )
+            .build()
+        )
+        .build();
+  }
+
+  private void initFreeFloatingEdgeAndRequest(
+    StreetMode mode,
+    RentalFormFactor formFactor,
+    boolean banNetwork
+  ) {
+    var network = "1";
+
+    VehicleRentalVehicle rentalVehicle = new VehicleRentalVehicle();
+
+    rentalVehicle.latitude = 1;
+    rentalVehicle.longitude = 1;
+    rentalVehicle.id = new FeedScopedId(network, "123");
+    rentalVehicle.vehicleType =
+      new RentalVehicleType(
+        new FeedScopedId(network, "type"),
+        "type",
+        formFactor,
+        RentalVehicleType.PropulsionType.ELECTRIC,
+        100000d
+      );
+
```

Comment:
Use TestFreeFloatingRentalVehicleBuilder for this.

Reply:
I ended up using the StreetModelForTest for this.